### PR TITLE
Create instructions for using the config file

### PR DESCRIPTION
**Pull request by seb231**
Config file instructions and usage for parameters

### CODE REVIEW

File: doc/config.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+The three required keys, `:random-seed`, `:simulations` and `:seed-year` define how the model will be run. `:modify-transitions-from` is optional and used in conjunction with `:transitions-parameters`.
+
+##### `:random-seed`
+
+Expects and integer with which to randomly seed the model with and enable repeatability.
+
+##### `:simulations`
+
+Expects an integer corresponding to a number with which to repeat the model over _n_ number of times. Due to parallelisation, the minimum number of simulations is eight. In tests 1000 simulations typically provides sufficient depth to normalise the results.
```
Comment (otfrom) [inline doc/config.md]: "Due to how we currently do parallelisation..." The minimum of 8 is a weird thing in how we do our code.

File: doc/config.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Expects a calendar year, provided as an integer with which to project the model from.
+
+##### `:modify-transitions-from`
+
+Expects a calendar year, provided as an integer, to start modifying transitions from, when `:modify-transition-by` & `:which-transitions?` are defined for either a [“Modify setting(s) transitions rates”](https://github.com/MastodonC/witan.send/blob/master/doc/scenarios.md#modify-settings-transitions-rates), [“Modify setting(s) transitions rates and transfer individuals to alternative setting(s)”](https://github.com/MastodonC/witan.send/blob/master/doc/scenarios.md#modify-settings-transitions-rates-and-transfer-individuals-to-alternative-settings) or [“Modify transitions from a specific future calendar year”](https://github.com/MastodonC/witan.send/blob/master/doc/scenarios.md#modify-transitions-from-a-specific-future-calendar-year) scenario.
+
+### `:output-parameters`
+
+Contains four required and two additional optional keys pertainting to what the model should output.
```
Comment (otfrom) [inline doc/config.md]: document what these are?

File: doc/config.md (hunk 1 above)
Comment (seb231) [reply doc/config.md]: Yes it is. I didn't want to change any process or code yet, just document how stuff is currently done.

File: doc/config.md (hunk 2 above)
Comment (seb231) [reply doc/config.md]: see below :)

**Pull request merged**